tests/bearlib/abstractions/external_bear_wrap_testfiles/test_external_bear.py

```diff
@@ -1,32 +1,46 @@
+import os
 import sys
 import json
 
 if __name__ == "__main__":
 
+    sys.path.append(os.path.join("..", "..", "..", ".."))
+
+    from coalib.results.Result import Result
+    from coalib.results.SourceRange import SourceRange
+    from coalib.output.JSONEncoder import create_json_encoder
+    from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
     line = sys.stdin.read()
     args = json.loads(line)
     settings = args['settings']
 
-    out = [{
-            'message': "This is wrong",
-            'file': args['filename'],
-            'severity': "MAJOR",
-            'line': 1
-        }, {
-            'message': "This is wrong too",
-            'file': args['filename'],
-            'severity': "INFO",
-            'line': 3}]
+    results = [
+        Result(
+            origin="TestBear",
+            message="This is wrong",
+            affected_code=(SourceRange.from_values(args['filename'], 1),),
+            severity=RESULT_SEVERITY.MAJOR),
+        Result(
+            origin="TestBear",
+            message="This is wrong too",
+            affected_code=(SourceRange.from_values(args['filename'], 3),),
+            severity=RESULT_SEVERITY.INFO)]
 
     if settings['a']:
-        for res in out:
-            res['severity'] = "NORMAL"
+        for res in results:
+            res.severity = RESULT_SEVERITY.NORMAL
 
     if settings['b']:
-        out[0]['debug_msg'] = "Sample debug message"
+        results[0].debug_msg = "Sample debug message"
 
     if not settings['c']:
```

Comment:
can you add a commit giving those proper names? a b and c doesn't tell me what they're doing and you could aid me understanding the code a lot by proper naming